### PR TITLE
Make more liquids evaporate

### PR DESCRIPTION
**Pull request by dvir001**
## About the PR

Add more liquids to evaporate system to combat lags

FluorosulfuricAcid
Vomit
InsectBlood
AmmoniaBlood
ZombieBlood
Blood
Slime
CopperBlood
Sap
JuiceTomato

## Why / Balance
After 6 hours its starting to hurt.

## How to test
Spawn blood, watch it dry.

## Media
N/A

## Breaking changes
Less work for janitors overall but worth the lag save.

**Changelog**
N/A

### CODE REVIEW

**Comment by dvir001** [comment]
> why lube? ...goodbye super slide =(((....make sure to add...tomato juice you soon find out why....also why does puddle of liquid creates lags?

Liquid calcs are wild

**Comment by whatston3** [comment]
Thoughts on adding a per-reagant rate in PuddleSystem.Evaporation so that these newer fluids evaporate more slowly?  It would also be neat if the non-water reagents left grime behind if they evaporated.

That should give some hint of a prior puddle for late forensics and give the janitor something to do for late-arriving spills.

I'll see if I can cook something up for this.

**Comment by Tomahawk1o1** [comment]
We have clean bots that can automatically go around turning these puddles to water to make them evaporate as well, could just add a few to frontier by default maybe?

**Comment by ThatOneGoblin25** [comment]
> We have clean bots that can automatically go around turning these puddles to water to make them evaporate as well, could just add a few to frontier by default maybe?

That only works on Frontier Station, and the additional AIs take up AI slots that expedition mobs could use.

**Comment by dvir001** [comment]
> Thoughts on adding a per-reagant rate in PuddleSystem.Evaporation so that these newer fluids evaporate more slowly? It would also be neat if the non-water reagents left grime behind if they evaporated.
> 
> That should give some hint of a prior puddle for late forensics and give the janitor something to do for late-arriving spills.
> 
> I'll see if I can cook something up for this. (edit: looks harder than I thought. If I can think of something cheeky, I'll follow up on this, but assume I won't.)

Changing the speed will require major edits to the code since its not codded around this (Hard coded) so im leaving it as it is.

**Comment by dvir001** [comment]
> yea can understand with...most common puddles like blood ect...but lube hurts =(

Replaced lube with JuiceTomato

File: Content.Shared/Fluids/SharedPuddleSystem.Evaporation.cs [hunk 1]
```diff
@@ -8,7 +8,18 @@ public abstract partial class SharedPuddleSystem
...
+    private const string FluorosulfuricAcid = "FluorosulfuricAcid"; // Frontier
+    private const string Vomit = "Vomit"; // Frontier
+    private const string InsectBlood = "InsectBloodt"; // Frontier
+    private const string AmmoniaBlood = "AmmoniaBlood"; // Frontier
+    private const string ZombieBlood = "ZombieBlood"; // Frontier
+    private const string Blood = "Blood"; // Frontier
+    private const string Slime = "Slime"; // Frontier
+    private const string CopperBlood = "CopperBlood"; // Frontier
+    private const string Sap = "Sap"; // Frontier
+    private const string JuiceTomato = "JuiceTomato"; // Frontier
```
Comment (whatston3) [inline Content.Shared/Fluids/SharedPuddleSystem.Evaporation.cs:20]: Shouldn't these have the same ValidatePrototypeId attribute added to them in the same way Water does?  Was this excluded for a reason?

For what it's worth, I'm not sure what impact, if any, that has at runtime, and I haven't been able to run the IntegrationTests.

```Csharp
    // Frontier: extend set of evaporating reagents (PR#1407)
    [ValidatePrototypeId<ReagentPrototype>]
    private const string FluorosulfuricAcid = "FluorosulfuricAcid";
    [ValidatePrototypeId<ReagentPrototype>]
    private const string Vomit = "Vomit";
    [ValidatePrototypeId<ReagentPrototype>]
    private const string InsectBlood = "InsectBlood";
    [ValidatePrototypeId<ReagentPrototype>]
    private const string AmmoniaBlood = "AmmoniaBlood";
    [ValidatePrototypeId<ReagentPrototype>]
    private const string ZombieBlood = "ZombieBlood";
    [ValidatePrototypeId<ReagentPrototype>]
    private const string Blood = "Blood";
    [ValidatePrototypeId<ReagentPrototype>]
    private const string Slime = "Slime";
    [ValidatePrototypeId<ReagentPrototype>]
    private const string CopperBlood = "CopperBlood";
    [ValidatePrototypeId<ReagentPrototype>]
    private const string Sap = "Sap";
    [ValidatePrototypeId<ReagentPrototype>]
    private const string JuiceTomato = "JuiceTomato";
    // End Frontier
```

File: Content.Shared/Fluids/SharedPuddleSystem.Evaporation.cs [hunk 2]
```diff
@@ -8,7 +8,18 @@ public abstract partial class SharedPuddleSystem
     [ValidatePrototypeId<ReagentPrototype>]
     private const string Water = "Water";
 
-    public static readonly string[] EvaporationReagents = [Water];
+    private const string FluorosulfuricAcid = "FluorosulfuricAcid"; // Frontier
+    private const string Vomit = "Vomit"; // Frontier
+    private const string InsectBlood = "InsectBloodt"; // Frontier
```
Comment (whatston3) [inline Content.Shared/Fluids/SharedPuddleSystem.Evaporation.cs:13]:
```suggestion
    private const string InsectBlood = "InsectBlood"; // Frontier
```

Typo?

**Review by whatston3 — CHANGES_REQUESTED** [review]
Tested, all solutions except insect blood evaporate - works after removing a typo.  Apart from a question with attribute use, no other issues here.

File: Content.Shared/Fluids/SharedPuddleSystem.Evaporation.cs (hunk 2 above)
Comment (dvir001) [reply Content.Shared/Fluids/SharedPuddleSystem.Evaporation.cs]: Yep

File: Content.Shared/Fluids/SharedPuddleSystem.Evaporation.cs (hunk 1 above)
Comment (dvir001) [reply Content.Shared/Fluids/SharedPuddleSystem.Evaporation.cs]: Trying to keep it simple even if its shit since its coming down from Upstream

**Pull request merged**